sdk/go/common/workspace/project.json

```diff
@@ -1,180 +1,307 @@
 {
-    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
The indent change in this file makes diff compare very hard :( can we revert to 4 space tab?